README.md

```diff
@@ -4,9 +4,8 @@
 [![Download](https://api.bintray.com/packages/avast/maven/rabbitmq-scala-client/images/download.svg) ](https://bintray.com/avast/maven/rabbitmq-scala-client/_latestVersion)
 
 
-This client is lightweight wrapper over standard [RabbitMQ java client](https://www.rabbitmq.com/java-client.html).
-It's API may be difficult to use for inexperienced RabbitMQ users. Goal of this library is to simplify basic use cases and shadow the programmer
-from the underlying client.
+This client is Scala-first wrapper over standard [RabbitMQ java client](https://www.rabbitmq.com/java-client.html). Goal of this library is
```

Comment:
What about removing the `javaapi`  stuff? I don't think that anyone will use it and it would reduce the code-base.

Reply:
Again? :smile:

Reply:
I agree that it does not make much sense to have a Scala library with finally tagless API and also with some other Java API. The only problem is that this used to be an internal library that is also used by Java developers so removing this would affect them (just because we decided to make this public and add finally tagless API to Scala) so that's why I would keep it for now (it's already there).